llvm/lib/Transforms/PeepholeOptimizationCourse/Func/Func.cpp

```diff
@@ -0,0 +1,41 @@
+#include "llvm/Transforms/PeepholeOptimizationCourse/Func.h"
+
+#include "llvm/IR/Function.h"
+#include "llvm/IR/Constants.h"
+#include "llvm/IR/IRBuilder.h"
+
+#include "llvm/Transforms/Utils/BasicBlockUtils.h"
+#include "llvm/IR/InstrTypes.h"
+
+namespace llvm {
+
+Instruction* ReplaceInt8Instruction(BinaryOperator* binOp) {
```

Comment:
The formula is incorrect

Reply:
Can you please specify what you mean by 'incorrect formula'?

Reply:
@ggladilov